optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -0,0 +1,1205 @@
+                "language_model": language_model_file_name,
+                "text_embeddings": text_embeddings_file_name,
+                "vision_embeddings": vision_embeddings_file_name,
+            }
+            for part in model_cls.additional_parts:
+                model_file_names[part] = part_file_name = f"openvino_{part}_model.xml"
+
+            file_names = model_file_names.copy()
+            for name, file_name in model_file_names.items():
+                model_cache_path = hf_hub_download(
+                    repo_id=model_id,
+                    filename=file_name,
+                    token=token,
+                    revision=revision,
+                    cache_dir=cache_dir,
+                    force_download=force_download,
+                    local_files_only=local_files_only,
+                )
+                file_names[name] = model_cache_path
+
+            model_save_dir = Path(model_cache_path).parent
+            language_model = model_cls.load_model(file_names["language_model"], quantization_config)
+            text_embeddings = model_cls.load_model(file_names["text_embeddings"], quantization_config)
+            vision_embeddings = model_cls.load_model(file_names["vision_emnbeddings"], quantization_config)
+            for part in model_cls.additional_parts:
+                kwargs[part] = model_cls.load_model(file_names[part], quantization_config)
+
+        try:
+            generation_config = GenerationConfig.from_pretrained(
+                model_id,
+                token=token,
+                revision=revision,
+                cache_dir=cache_dir,
+                force_download=force_download,
+                local_files_only=local_files_only,
+            )
+            kwargs["generation_config"] = generation_config
+        except Exception:
+            pass
+
+        return model_cls(
+            language_model=language_model,
+            text_embeddings=text_embeddings,
+            vision_embeddings=vision_embeddings,
+            config=config,
+            model_save_dir=model_save_dir,
+            quantization_config=quantization_config,
+            **kwargs,
+        )
+
+    @classmethod
+    def _from_transformers(
+        cls,
+        model_id: str,
+        config: PretrainedConfig,
+        use_auth_token: Optional[Union[bool, str]] = None,
+        token: Optional[Union[bool, str]] = None,
+        revision: Optional[str] = None,
+        force_download: bool = False,
+        cache_dir: str = HUGGINGFACE_HUB_CACHE,
+        subfolder: str = "",
+        local_files_only: bool = False,
+        task: Optional[str] = None,
+        use_cache: bool = True,
+        trust_remote_code: bool = False,
+        load_in_8bit: Optional[bool] = None,
+        quantization_config: Union[OVWeightQuantizationConfig, Dict] = None,
+        **kwargs,
+    ):
+        """
+        Export a vanilla Transformers model into an ONNX model using `transformers.onnx.export_onnx`.
```

Comment:
Do we really export to the ONNX? The docstrings should be revised accordingly.

Reply:
I accidentally copied that from https://github.com/huggingface/optimum-intel/blob/main/optimum/intel/openvino/modeling_base.py#L534
I see the same doc string in other model classes for openvino (seq2seq, stable diffusion support e.t.c.), @echarlaix  maybe it is time to revise them in all models too?